samples/python/whisper_speech_recognition/whisper_speech_recognition.py

```diff
@@ -20,27 +18,26 @@ def main():
 
     raw_speech = read_wav(args.wav_file_path)
 
-    pipe = openvino_genai.WhisperPipeline(args.model_dir)
+    device = "CPU" # can switch between CPU/GPU
+    pipe = openvino_genai.WhisperPipeline(args.model_dir, device=device) 
```

Comment:
Can you update C++ version in the same way to keep them aligned?

Reply:
I have to spin up the C++ and update after. Maybe we will merge this first, and will work on C++ on the next PR?